azurerm/internal/services/policy/data_source_policy_definition.go

```diff
@@ -23,17 +23,21 @@ func dataSourceArmPolicyDefinition() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"display_name": {
 				Type:         schema.TypeString,
-				Required:     true,
+				Optional:     true,
+				Computed:     true,
 				ValidateFunc: validation.StringIsNotEmpty,
+				ExactlyOneOf: []string{"name", "display_name"},
 			},
-			"management_group_id": {
+			"name": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: azure.ValidateResourceIDOrEmpty,
+				Computed:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+				ExactlyOneOf: []string{"name", "display_name"},
 			},
-			"name": {
+			"management_group_id": {
```

Comment:
could we deprecate this to `management_group_name` to match #6276

Reply:
Done